content/api-streamer/consumer.textile

```diff
@@ -0,0 +1,34 @@
+---
+title: Consumer
+section: api-streamer
+index: 68
+---
+
+API Streamer has two roles: a producer and consumer. A consumer is usually invited to API Streamer by a producer so the consumer can subscribe to, manage, and consume the data streams. A data stream could be anything from live transport updates to a stream of sensor data from an embedded IoT device.
```

Comment:
> A consumer is usually invited to API Streamer by a producer so the consumer can subscribe to, manage, and consume the data streams

I don't like this "usually invited" part.  What happens the rest of the time.

"A consumer subscribes to streams offered by producers using any of the many open push and pull protocols supported by Ably."  I am not really sure how subscribe/consume is different, or why management is important for consumers